tests/code/test_transcripts.py

```diff
@@ -0,0 +1,57 @@
+import pytest
+from peewee import DoesNotExist
+
+from app.controllers.events.view_transcript import getSLCourseTranscript
+from app.controllers.events.view_transcript import getProgramTranscript
+from app.models.term import Term
+from app.models.course import Course
+from app.models.courseParticipant import CourseParticipant
+from app.models.user import User
+from app.models.eventParticipant import EventParticipant
+
+@pytest.mark.integration
+def test_noGetSLTranscript():
+
+    with pytest.raises(DoesNotExist):
+        sLcourse = getSLCourseTranscript("ggiuo")
+    with pytest.raises(DoesNotExist):
+        sLcourse = getSLCourseTranscript(69809)
+
+
+@pytest.mark.integration
+def test_noGetProgramTranscript():
+
+    with pytest.raises(DoesNotExist):
+        sLcourse = getProgramTranscript("adfafa")
+    with pytest.raises(DoesNotExist):
+        sLcourse = getProgramTranscript(56498)
+
+@pytest.mark.integration
+def test_getSLTranscripts():

```

Comment:
It's a good idea to test the empty case and make sure there isn't an error. Like, for a valid user without any courses do you just get an empty array like you expect?

Reply:
we added a test for a user who doesn't have any cases and it returns an empty array.